docs/apis-tools/operate-api/authentication.md

```diff
@@ -4,57 +4,141 @@ title: Authentication
 description: "Authentication requirements for accessing the Operate REST API."
 ---
 
-All Operate REST API requests require authentication.
-
-## Authentication for SaaS
-
-### Authentication via JWT access token
-
-You must pass an access token as a header in each request to the SaaS Operate API. When you create an Operate [client](/guides/setup-client-connection-credentials.md), you get all the information needed to connect to Operate.
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+All Operate REST API requests require authentication. To authenticate, generate a JWT token and pass it in each request.
+
+## Generating a token
+
+<Tabs groupId="authentication" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+<TabItem value='saas'>
+
+1. [Create client credentials](/guides/setup-client-connection-credentials.md) in the **Clusters > Cluster name > API** tab of [Camunda Console](https://console.camunda.io/).
+2. Add permissions to this client for **Operate**.
+3. Upon creating the client, capture the following values required to generate a token:
+   <!-- this comment convinces the markdown processor to still treat the table as a table, but without adding surrounding paragraphs. 🤷 -->
+   | Name                     | Environment variable name        | Default value                                |
+   | ------------------------ | -------------------------------- | -------------------------------------------- |
+   | Client ID                | `ZEEBE_CLIENT_ID`                | -                                            |
+   | Client Secret            | `ZEEBE_CLIENT_SECRET`            | -                                            |
+   | Authorization Server URL | `ZEEBE_AUTHORIZATION_SERVER_URL` | `https://login.cloud.camunda.io/oauth/token` |
+   | Audience                 |                                  | `operate.camunda.io`                         |
+   | Operate REST Address     | `CAMUNDA_OPERATE_BASE_URL`       | -                                            |
+   <!-- this comment convinces the markdown processor to still treat the table as a table, but without adding surrounding paragraphs. 🤷 -->
+   :::tip
```

Comment:
NBC: Should this be a caution instead of a tip as it is quite important as they cannot get the secret again if they don't copy it now.

Reply:
I don't feel strongly either way about this; if you do, let me know and I'll adjust.